tests/test_visualizer.py

```diff
@@ -0,0 +1,49 @@
+"""
+Tests for the visualizer module
+"""
+
+import pytest
+from matplotlib import pyplot as plt
+import numpy as np
+import os
+import pandas as pd
+from pathlib import Path
+
+from ppinsight import visualizer
+
+# setup variables
+models = ['../dummy_scores/model_1.csv','/home/fmclary/CSE583/PPInsight/dummy_scores/model_2.csv','/home/fmclary/CSE583/PPInsight/dummy_scores/model_3.csv']
```

Comment:
Not relevant for this HW and you don't have to make this change, but I'd either:

1) move `models` to a `config.py` or `test_config.py` file 
2) modify this to `MODELS`, which signifies a constant. It's just a python convention

Also, gotta make sure you don't have `/home/fmclary/CSE583/PPInsight/` in the final product. I mentioned this in the github issue I opened on this repo, but I'm commenting directly so you know where to look